src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -86,134 +101,148 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         if (false !== strpos($input->getFirstArgument(), ':d')) {
-            $output->writeln('<comment>The use of "translation:debug" command is deprecated since version 2.7 and will be removed in 3.0. Use the "debug:translation" instead.</comment>');
+            $output->writeln(
+                '<comment>The use of "translation:debug" command is deprecated since version 2.7 and will be removed in 3.0. Use the "debug:translation" instead.</comment>'
+            );
         }
 
         $locale = $input->getArgument('locale');
         $domain = $input->getOption('domain');
-        $loader = $this->getContainer()->get('translation.loader');
-        $kernel = $this->getContainer()->get('kernel');
+        /** @var TranslationLoader $loader */
+        $loader = $this->getContainer()
```

Comment:
Why add line break here ? we can do `$loader = $this->getContainer()->get('translation.loader');`